PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -258,32 +285,51 @@ class PayPalClient internal constructor(
 
     private fun callbackBrowserSwitchCancel(
         callback: PayPalTokenizeCallback,
-        cancel: PayPalResult.Cancel
+        cancel: PayPalResult.Cancel,
+        isAppSwitchFlow: Boolean
     ) {
         braintreeClient.sendAnalyticsEvent(PayPalAnalytics.BROWSER_LOGIN_CANCELED, analyticsParams)
+
+        if (isAppSwitchFlow) {
+            braintreeClient.sendAnalyticsEvent(PayPalAnalytics.APP_SWITCH_FAILED, analyticsParams)
```

Comment:
Would we want to track the canceled flow separately from `APP_SWITCH_FAILED`?

Reply:
Agree, but for now, I think we're only requiring the events that are currently added. We can always iterate later. What do you think?

Reply:
I think we should try to have our events in a final state before we do an SDK release. It'll be hard to sift through the data if we end up changing this to a unique Cancel event in a later release. Can we sync with the larger team to see if we can add a cancel event for app switch?

Reply:
Thanks you, updated: [ae501ff](https://github.com/braintree/braintree_android/pull/1111/commits/ae501ff361186528495f31308b1ba36b012147d1)